signac_dashboard/templates/cards/logger.html

```diff
@@ -0,0 +1,50 @@
+<div>
+    <input type="checkbox" name="follow" id="follow-{{jobid}}" onchange="logger_{{jobid}}.setFollow()">
```

Comment:
This won't work if multiple loggers are present. The ids should include the job id and some other unique identifier. Also see the examples of the Notes and DocumentEditor modules for how to include custom JavaScript. The JS shouldn't be included repeatedly for each job id / module instance.